src/types/lib.ts

```diff
@@ -77,6 +77,21 @@ export type InvocationsDetails = {
 
 export type InvocationsDetailsWithNonce = InvocationsDetails & { nonce: BigNumberish };
 
+export type TransactionBulk = Array<
+  { type: TransactionType } & {
+    payload: DeclareContractPayload | AllowArray<Call> | DeployAccountContractPayload;
+  }
```

Comment:
Needs better typing. This can allow passing `INVOKE_FUNCTION` with DeclareContractPayload arguement